pingdom-checks/community-care.ping

```diff
@@ -12,7 +13,7 @@ pingdom save-check \
   -a group="community-care" \
   -a authorization_token="$HEALTH_APIS_STATIC_ACCESS_TOKEN" \
   -a userids_csv="$STATUSPAGE_PRODUCTION_COMMUNITY_CARE_ELIGIBILITY" \
-  -a integrationids_csv="$COMMUNITY_CARE_SLACK_ID,$PAGER_DUTY"
+  -a integrationids_csv="$COMMUNITY_CARE_SLACK_ID,$COMMUNITY_CARE_ALARMS_SLACK_ID,$PAGER_DUTY"
```

Comment:
These names need to be tweaked, reading this line it's not clear what is the difference between `COMMUNITY_CARE_SLACK_ID` and `COMMUNITY_CARE_ALARMS_SLACK_ID`